src/js/base/stories/BoxStories/BoxTitle.stories.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { BoxTitle, BoxGroup } from "../../Box";
+
+export default {
+    title: "base/Box/BoxTitle",
+    component: BoxTitle
+};
+
+const Template = args => (
+    <BoxGroup>
+        <BoxTitle {...args} />
+        <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+            dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex
+            ea commodo consequat.
+        </p>
+    </BoxGroup>
```

Comment:
```suggestion
    <Box>
        <BoxTitle {...args} />
        <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat.
        </p>
    </Box>
```

This component still looks a bit funny. Pretty sure that is because `BoxTitle` meant to be used as a child of a `Box` rather than `BoxGroup`.